fe/src/main/cup/sql_parser.cup

```diff
@@ -597,6 +598,17 @@ alter_stmt ::=
     :}
     ;
 
+quota_expr ::=
```

Comment:
quota_expr is not a good name, because everything whose unit is byte can use this like exec_mem_limit

can you change it?